code/modules/antagonists/blob/blob_mobs.dm

```diff
@@ -127,7 +127,7 @@
 /mob/living/simple_animal/hostile/blob/blobspore/Life()
 	if(!is_zombie && isturf(src.loc))
 		for(var/mob/living/carbon/human/H in view(src,1)) //Only for corpse right next to/on same tile
-			if(H.stat == DEAD)
+			if(!istype(src, /mob/living/simple_animal/hostile/blob/blobspore/weak) && H.stat == DEAD)
```

Comment:
if you do this, would you mind removing weak blob spores from the xenobio gold slime core pool so that they don't become an absolute dud result? keep the normal blob spores in there, of course; they're cool.

Reply:
Sure.

Reply:
Don't use `istype(src)`
Either redefine the proc or set up proper variables/flags for this kind of behavior.